test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -759,6 +759,14 @@ def forward(self, x):
         x = torch.randn(4, 2, 3, requires_grad=True)
         self.run_test(NormModel(), x)
 
+    def test_gelu(self):
```

Comment:
If all ops used for composition is supported, could we also add a test for Caffe2?

Reply:
Sure, done